prioritize.py

```diff
@@ -196,6 +380,12 @@ def main(
     We embedded the assets using a generic model, and then use a strategy that is a mixture of
     diversity sampling, uncertainty sampling, and random sampling to sorts the assets.
     """
+    if uncertainty_sampling + diversity_sampling > 1:
+        raise ValueError("diversity_sampling + diversity_sampling should be less than 1.")
+
+    if max_assets < 10:
```

Comment:
max assets can be None (this is a type bug to fix in the main definition, my bad)